apps/docs/src/development/index.md

```diff
@@ -53,3 +59,80 @@ import { OnyxButton } from "sit-onyx";
   <OnyxButton label="Click me" />
 </template>
 ```
+
+You can check all onyx components and their documentation in our [Storybook](https://storybook.onyx.schwarz).
+
+## First steps <Badge text="recommended" />
+
+Here are some further steps we recommend to take a look at when setting up your new application:
+
+### Layout components
+
+onyx provides [layout components](https://storybook.onyx.schwarz/?path=/docs/layout-about-layouts--docs) for easily setting up common layouts for the app, pages etc. which also take care of ensuring correct scroll containers etc.
+
+Here is an example how a default application looks like when using layout components:
+
+```vue
+<script lang="ts" setup>
+import { OnyxAppLayout, OnyxPageLayout, OnyxNavBar } from "sit-onyx";
+</script>
+
+<template>
+  <OnyxAppLayout>
+    <template #navBar>
+      <OnyxNavBar app-name="My app" />
+    </template>
+
+    <OnyxPageLayout>
+      <div class="onyx-grid-container">Your page content here...</div>
+    </OnyxPageLayout>
+  </OnyxAppLayout>
+</template>
+```
+
+### Customize CSS
+
+For most cases, all onyx components already provide properties to easily use supported features.
```

Comment:
```suggestion
For most cases, all onyx components already provide properties to easily use supported features that are shown in an onyx Figma design.
```

Reply:
The features does not have to be shown in Figma for it to be supported?^^